src/persist/EpochSys.hpp

```diff
@@ -115,6 +115,12 @@ class PBlk{
     inline bool retired(){
         return retire != nullptr;
     }
+    inline void mark_epoch_container(){
```

Comment:
It's for assigning epoch container a special id `~0x0ULL`. I added this when debugging the missing epoch container bug: epoch container is never assigned an id, so it's initialized to 0, which happens to be the id of the first allocated PBlk on thread 0 assigned by UIDGnererator. Recovery may decide that they share the same ID and accidentally deleted the container.

While I later figured out this is  _not_ the root cause of the bug, I still figured it would be nice to assign the container an ID that is unlikely to conflict with any other PBlks.